code-blast.js

```diff
@@ -205,9 +205,9 @@ https://twitter.com/JoelBesada/status/670343885655293952
 		}
 
 		function onCodeMirrorChange(e) {
-			throttledShake(0.3);
-
 			setTimeout(function() {
+				if (!e) e = { start: editor.getCursorPosition() };
```

Comment:
I wanted to add a public method to allow triggering shake when the option is activated https://github.com/leMaik/code-blast-ace/pull/4/files#diff-4730e00fd5d64cae591e32dd42da6a10R257
For use case like https://github.com/c9/core/issues/386#issuecomment-294366386